packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme.dart

```diff
@@ -21,9 +19,9 @@ class IdeTheme {
     this.embed = false,
   });
 
-  final Color backgroundColor;
-  final Color foregroundColor;
-  final double fontSize;
+  final Color? backgroundColor;
+  final Color? foregroundColor;
+  final double? fontSize;
```

Comment:
make font size non-nullable and use unscaledDefaultFontSize as a default where we are currently passing in null for fontSize

Reply:
done